src/main/java/storagesystem/viewcontroller/settings/teamPage.fxml

```diff
@@ -40,6 +40,31 @@
             <Text fill="WHITE" layoutX="64.0" layoutY="203.0" strokeType="OUTSIDE" strokeWidth="0.0" styleClass="bigLabel" text="Team Name:" underline="true" wrappingWidth="150.00000596046448" AnchorPane.topAnchor="180.0" />
```

Comment:
Mouse changes on hover, which indicates you can edit it which you can't. Mouse shouldn't change